README.rst

```diff
@@ -68,9 +68,11 @@ Windows
     <your-env>\Scripts\activate
     <your-env>\Scripts\pip.exe install google-cloud-video-transcoder
 
+
 Next Steps
 ~~~~~~~~~~
 
+-  See the [Samples](./samples/snippets/README.md).
```

Comment:
Nit: this link doesn't seem to show up properly, but feel free to address this in a different PR.